### ISSUE TITLE
Icon: Doesn't consume 'aria-labelledby' from ImageProps

### ISSUE THREAD

**Issue by HalcyonPride** [OfficeDev/office-ui-fabric-react#9997]
### Environment Information 
 
- __Package version(s)__: 7.18.1
- __Browser and OS versions__: N.A.

### Please provide a reproduction of the bug in a codepen:

https://codepen.io/anon/pen/xvdEPY
 

 
#### Actual behavior:
 
Nothing.
 
#### Expected behavior:
 
`aria-labelledby` - and similar HTML attributes - in `imageProps` should be applied to the Icon's `<i>` element.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? **No**
 
Requested priority: **Normal; potentially an accessibility issue**
 
Products/sites affected: **N.A.**

**Comment by marygans** [comment]
Hi @HalcyonPride thanks for submitting this and referencing the codePen example! I was able to repro this and will work on getting it fixed.

**Comment by ecraig12345** [comment]
Noticed this issue since I'm also working on some Icon changes (#9884). I'm not sure that hoisting `aria-*` attributes from `imageProps` to the icon root makes sense. If you want those attributes applied to the root, you can apply them there directly (see https://codepen.io/ecraig12345/pen/WNewvMd?editors=0010). Is there a case where you're not able to apply the attributes to the root for some reason?

**Comment by HalcyonPride** [comment]
@ecraig12345, I can't say - I'm just basing my comments off what's already written in Icon.base.tsx:
![image](https://user-images.githubusercontent.com/8681794/63064007-0fcd1500-beb4-11e9-9632-bed9c0120039.png)

**Comment by ecraig12345** [comment]
All that's doing is determining whether the container should be `aria-hidden` based on the provided `imageProps`. It's not directly passing any props through.

**Comment by HalcyonPride** [comment]
Yes, but it's implying that `aria-labelledby` etc. is passed via `imageProps`. Otherwise, why is it trying to read those attributes from `imageProps`?

Conversely, this code needs to be updated if we expect `aria-*` attributes to be passed via the icon root instead.

**Comment by HalcyonPride** [comment]
@dzearing, that sounds good to me 😊 thanks for the feedback!
@leddie24, could you please revise your code per David's proposal?
